lib/sen_tweet_web/controllers/page_controller.ex

```diff
@@ -1,7 +1,12 @@
 defmodule SenTweetWeb.PageController do
   use SenTweetWeb, :controller
-
+  # import SenTweet.Bitfeels
   def index(conn, _params) do
     render(conn, "index.html")
   end
+
+  def stream_admin(conn, _params) do
+    streams = Bitfeels.all_streams()
```

Comment:
`:bitfeels` is an application that we add as a part of our `deps`, just like the `:phoenix` application. With the `:bitfeels` application running we have access to the [Bitfeels](https://github.com/thekeele/bitfeels/blob/master/lib/bitfeels.ex) module. This module is the interface to interact with the streams.